modules/tabs/main/index.coffee

```diff
@@ -1,36 +1,116 @@
-onTabSelected = (tabs, element, i) ->
+onTabSelected = (tabs, element, i, cause) ->
   tabs.selected = i
   selection = hx.select(tabs.selector)
   selection.selectAll('.hx-tab').classed('hx-tab-active', false)
   hx.select(element).classed('hx-tab-active', true)
 
   context = hx.palette.context(element)
-  hx.palette.borderContext(selection.select('.hx-tabs-content'), context)
+  tabsContent = selection.select('.hx-tabs-content')
+  hx.palette.borderContext(tabsContent, context)
 
-  hx.select(tabs.selector).selectAll('.hx-tab-content').classed('hx-tab-content-hidden', true)
-  hx.select('#' + hx.select(element).attr('data-content')).classed('hx-tab-content-hidden', false)
```

Comment:
Although element IDs should be unique in the document, this won't work if the tabs are in a detached div